README.md

```diff
@@ -179,52 +179,6 @@ journaling. Note that the less-frequently-used methods `toggle`,
 `increment*`, `decrement*`, and `update_counters` are not intercepted at
 this time.
 
-### Tagged Events
```

Comment:
I reorganized the README, because I realized that "tagged events" needed to go below "custom journaling"